Change the continuous link

SeanCurtis-TRI opened the pull request:
The overview links to "Continuous Production".  The body links to
"Continuous".  Now they both link to "Continuous Production".  This will
help new build cops to acclimate better.

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/robotlocomotion/drake/4051)
<!-- Reviewable:end -->

SeanCurtis-TRI commented:
+@david-german-tri for feature reivew +@sherm1 for platform review.   PTAL

---

Review status: 0 of 1 files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/4051#-:-KVzvyAbid7LKM99djQD:b-gvlfv6)*
<!-- Sent from Reviewable.io -->

david-german-tri commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/> 

---

Review status: 0 of 1 files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/4051#-:-KVzw_YnTwMn-jF3lPv6:bg7igkv)*
<!-- Sent from Reviewable.io -->

SeanCurtis-TRI commented:
I think I finally found a PR that I can push through without debate.  :)

---

Review status: 0 of 1 files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/4051#-:-KVzwjZBsIRGaK7TRfVu:buh3q28)*
<!-- Sent from Reviewable.io -->

liangfok commented:
Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

*[drake/doc/buildcop.rst, line 124 at r1](https://reviewable.io:443/reviews/robotlocomotion/drake/4051#-KW--Fq28ljdFuRxyPSU:-KW--Fq28ljdFuRxyPSV:bs5yvtn) ([raw file](https://github.com/robotlocomotion/drake/blob/294ca3696b76ce9bd7145f0f30c318367bbaf1ac/drake/doc/buildcop.rst#L124)):*
> ```reStructuredText
> ^^^^^^^^^^^^^^^^^
> Check the `Continuous Production <https://drake-
> jenkins.csail.mit.edu/view/Continuous%20Production/>`_ 
> ```

BTW, have you verified this looks good in the generated `Sphinx`-based website? Just wanted to make sure the HTML link generates correctly when it's split into two lines.

Also, looks like there's an extra space after the link.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/4051)*
<!-- Sent from Reviewable.io -->

SeanCurtis-TRI commented:
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion.

---

*[drake/doc/buildcop.rst, line 124 at r1](https://reviewable.io:443/reviews/robotlocomotion/drake/4051#-KW--Fq28ljdFuRxyPSU:-KW-4isoMp4PcBFQUVfJ:b-ucdd93) ([raw file](https://github.com/robotlocomotion/drake/blob/294ca3696b76ce9bd7145f0f30c318367bbaf1ac/drake/doc/buildcop.rst#L124)):*
<details><summary><i>Previously, liangfok (Chien-Liang Fok) wrote…</i></summary>
> BTW, have you verified this looks good in the generated `Sphinx`-based website? Just wanted to make sure the HTML link generates correctly when it's split into two lines.
> 
> Also, looks like there's an extra space after the link.
</details>
It does build and link properly.  However, for the sake of peace of mind, I've modified the line break and removed the stray whitespace.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/4051)*
<!-- Sent from Reviewable.io -->

liangfok commented:
Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/4051)*
<!-- Sent from Reviewable.io -->

liangfok commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

---

Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/4051#-:-KW-5Xh69sDTXOdNasfa:bnfp4nl)*
<!-- Sent from Reviewable.io -->

SeanCurtis-TRI commented:
@david-german-tri Somehow it is looking to you for platform LGTM.  Even though @sherm1  is the platform reviewer, you want to rubber stamp this and I'll remove him?

---

Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/4051#-:-KW-ML7CJyTDFJ-UAqOP:blhc2ax)*
<!-- Sent from Reviewable.io -->

liangfok commented:
I believe we're waiting for an LGTM from @sherm1.

---

Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/4051#-:-KW-N936h4bU9JAMOP22:b-rxx929)*
<!-- Sent from Reviewable.io -->

liangfok commented:
I believe if you remove @sherm1 as an assigned reviewer, you can merge.

---

Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/4051#-:-KW-NK5t4m9X1B7EuxFw:b-8tuktp)*
<!-- Sent from Reviewable.io -->

SeanCurtis-TRI commented:
That's what I would've thought, but the Reviewable "Checks" indicates the following: 

![✂-1](https://files.reviewableusercontent.io/images/reviewable/22159799/jWjTVU-oXlxsRc7b5UMpbIBD.png)

---

Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/4051#-:-KW-NNBTH6zDpoEW_pzR:b-qz6h46)*
<!-- Sent from Reviewable.io -->

liangfok commented:
I believe the text after the comma indicate @david-german-tri has already provided a platform LGTM. In other words, I believe "missing" only applies to the text before the comma...

---

Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/4051#-:-KW-Nr4vBUt-33lL7WKa:b-x1l3wy)*
<!-- Sent from Reviewable.io -->

david-german-tri commented:
What @liangfok said!

---

Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/4051#-:-KW-SqidhJXZ7B8YQFm-:b-yb0fq2)*
<!-- Sent from Reviewable.io -->

SeanCurtis-TRI commented:
Wow....punctuation has been put through the wringer.  Well, since @sherm1 seems swamped and this is so small (and has been looked at by two people), I'm going to remove him. -@sherm1 

---

Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/4051#-:-KW-WsilE-JtWx2gnVVZ:bxzsibn)*
<!-- Sent from Reviewable.io -->
